presentation/src/main/res/navigation/nav_graph.xml

```diff
@@ -3,26 +3,12 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:id="@+id/nav_graph"
-    app:startDestination="@id/FirstFragment">
+    app:startDestination="@id/QuestionFragment">
 
     <fragment
-        android:id="@+id/FirstFragment"
-        android:name="com.martgant.tolkienquizapp.FirstFragment"
-        android:label="@string/first_fragment_label"
-        tools:layout="@layout/fragment_first">
-
-        <action
-            android:id="@+id/action_FirstFragment_to_SecondFragment"
-            app:destination="@id/SecondFragment" />
-    </fragment>
-    <fragment
-        android:id="@+id/SecondFragment"
-        android:name="com.martgant.tolkienquizapp.SecondFragment"
```

Comment:
You still have `SecondFragment` class in src/main/java - because of it it's not compiling because of unresolved reference  to`action_SecondFragment_to_FirstFragment`  (which you just removed). 
You also can remove `fragment_second`

Reply:
Of course I meant you can remove `SecondFragment`